frame/support/test/tests/construct_runtime_ui/no_std_genesis_config.stderr

```diff
@@ -0,0 +1,66 @@
+error: `Pallet` does not have the std feature enabled, this will cause the `PalletConfig` type to be undefined.
```

Comment:
I think the actual type is not `PalletConfig` but `test_pallet::GenesisConfig`. Having the real type makes error message more understandable to me.

Reply:
The "will" here actually signifies something that's expected or required to happen, instead of being a [modal auxiliary](https://en.wikipedia.org/wiki/Modal_verb) for the future tense.

As for the path, I thought about that before, but is it possible to stringize a path fragment in a declarative macro? i.e. given that we have

```rust
macro_rules! foo {
    ($bar: path) => // What should go here to make a &'static str out of `$bar`?
}
```

Do I just use the `stringify!` macro for this?

Reply:
you can give the string to the macro directly considering it is called from a proc_macro where you can easily build some string literal